### DISCUSSION TITLE
My web site loses formatting after uploading

### DISCUSSION THREAD

**Discussion by PapaTuck** [community/community#22089] [Other Feature Feedback, Questions, & Ideas]
<p>I am new to coding and have created a web page using HTML and CSS. It looks good locally in 3 different browsers but when I upload the files here my page loses formatting -fonts, colors, alignment. Everything becomes left-aligned.</p>
<p>Also, I pasted the code into CodePen and it looked good there too.</p>
<p>Any help would be appreciated!</p>

**Comment by nyahbhinghiPrincess** [comment]
<p>Hello <a class="mention" href="/papatuck">@PapaTuck</a>! Welcome to the community; we are glad to have you here.  Can you tell us more about the overall setup or file structure of your CSS and HTML files in that new repository?</p>
<p>If it is a public repo and you are comfortable sharing - then you can share that link here for eyes from the community.  In general, you will want to be sure that your CSS is being called from the correct location.  Here is a previous example on the forum, although it may not be the same issue in your case: <a href="https://github.community/t/codepen-to-github-css-not-applying/119076" class="inline-onebox">Codepen to Github CSS not applying</a></p>

**Comment by PapaTuck** [comment]
<p>Thanks for the info and for getting back with me. I’ll be putting the site back up shortly. I’ll check out the CSS link and see if that does the trick. If not, I’ll share a link.</p>

**Comment by PapaTuck** [comment]
<aside class="onebox whitelistedgeneric">
  <header class="source">
      <img src="https://github.githubassets.com/favicons/favicon.svg" class="site-icon" width="32" height="32">
      <a href="https://github.com/PapaTuck/papatuck.github.io" target="_blank" rel="nofollow noopener">GitHub</a>
  </header>
  <article class="onebox-body">
    <img src="https://user-images.githubusercontent.com/63917642/181068837-8f67afd3-063f-461a-a31f-18e8f90eeb0f.png" class="thumbnail onebox-avatar" width="360" height="360">

<h3><a href="https://github.com/PapaTuck/papatuck.github.io" target="_blank" rel="nofollow noopener">PapaTuck/papatuck.github.io</a></h3>

<p>My Site page -HTML and CSS. Contribute to PapaTuck/papatuck.github.io development by creating an account on GitHub.</p>


  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

**Comment by nyahbhinghiPrincess** [comment]
<p>Thanks so much for sharing that.  I noticed that <a href="https://papatuck.github.io/css/styles.css">https://papatuck.github.io/css/styles.css</a> loads with a 404 but <a href="https://papatuck.github.io/CSS/styles.css">https://papatuck.github.io/CSS/styles.css</a> shows a listing of CSS.</p>
<p>You may want to the change the uppercase/lowercase matching between the styles.css folder name and the href link in the  tag of your index.html file. Does that help?</p>

**Reply by mirshad47** [reply]
can u help me caz i'm also facing the same problem.
my live site link:
https://mirshad47.github.io/g3-architects-assingment/

https://github.com/mirshad47/g3-architects-assingment

**Comment by PapaTuck** [comment, marked as answer]
<p>:+1: Absolutely! I didn’t notice the folder name case being different than the link.</p>

**Comment by Vaishnavi-3601** [comment]
Im also facing the problem in which my website moved slightly to the left 
https://vaishnavi-3601.github.io/Simsun/
